### PR TITLE
Change AppPackager call parameter attributes to avoid :

### PR DESCRIPTION
**Pull request by rbarros-daitan**
Change ":" to "=" on AppPackager call..

### CODE REVIEW

**Comment by rbarros-daitan** [comment]
@lneir , can you please review? Thanks!

File: installer/Symphony-x86.aip [hunk 1]
```diff
@@ -384,7 +384,7 @@
     <ROW Action="KillParagon_1" Type="1" Source="aicustact.dll" Target="StopProcess" AdditionalSeq="AI_DATA_SETTER_3"/>
     <ROW Action="KillRenderer" Type="1" Source="aicustact.dll" Target="StopProcess" Options="1" AdditionalSeq="AI_DATA_SETTER"/>
     <ROW Action="KillRenderer_1" Type="1" Source="aicustact.dll" Target="StopProcess" AdditionalSeq="AI_DATA_SETTER_1"/>
-    <ROW Action="Paragon.AppPackager.exe" Type="3602" Source="Paragon.AppPackager.exe" Target="--u:[POD_URL] &quot;--p:[#Symphony.pgx]&quot;"/>
+    <ROW Action="Paragon.AppPackager.exe" Type="3602" Source="Paragon.AppPackager.exe" Target="&quot;--u=[POD_URL]&quot; &quot;--p=[#Symphony.pgx]&quot;"/>
```
Comment (lneir) [inline installer/Symphony-x86.aip:387]: i thought change was also needed to AppPackager?

Comment (lneir) [reply installer/Symphony-x86.aip:387]: also why add the &quot and semicolon to front of Target? other after [POD_URL]

Comment (rbarros-daitan) [reply installer/Symphony-x86.aip:387]: no, change is only on installer. This way I avoided changing AppPackager so less tests are needed. &quot is to have same pattern on both parameters.

**Review by lneir — APPROVED** [review]

**Pull request merged**